client/src/components/shared/ModalContent.component.js

```diff
@@ -30,25 +31,29 @@ export default class ModalContent extends Component {
   renderListingDetails() {
     return (
       <div id="restaurant-address">
-        <h3 className="m-3 mb-4">Donation details</h3>
-        <p>
-          There are {this.props.content.listingNumberOfMeals} meals available
-        </p>
-        <p>Details below: </p>
-        <p>{this.props.content.listingDescription}</p>
-        <p>
-          Grab this donation before {this.props.content.listingAvailableTill}
+        <p class="font-weight-bold">
+          There are {this.props.content.listingNumberOfMeals} meals available to
+          collect until {this.props.content.listingAvailableTill}.
```

Comment:
Replace with this to get Lizzy's formatting `{Date(this.props.content.listingAvailableTill)}`